src/utilities/FormManager.ts

```diff
@@ -197,8 +197,9 @@ class FormManager {
     });
   }
 
-  private hasErrors (fieldsError: any) {
-    return Object.keys(fieldsError).some((field) => fieldsError[field]);
+  private hasErrors () {
+    const fieldsError = flattenObject<{ [key: string]: any }, { [key: string]: any }>(this.form.getFieldsError());
```

Comment:
kind of weird to read, but i suppose this is ok since it's taken from code earlier in the file